code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1005,7 +1005,7 @@
 	// Allows you to put in item-specific reactions based on species
 	if(user != H)
 		user.do_attack_animation(H)
-	if(H.check_shields(I.force, "the [I.name]", I, 0, I.armour_penetration))
+	if(user != H && H.check_shields(I.force, "the [I.name]", I, 0, I.armour_penetration))
```

Comment:
@duncathan 

It's a bug. I was the one who added the armour originally (now you know why its so shitty and snowflake!) and it was never intended behaviour.

We also can't have any other armours (like spitting fire) if you can run around hitting yourself with a medkit to self trigger, it's needed for the refactor.